TODO/build-tic-tac-toe-with-ai-using-swift.md

```diff
@@ -1,50 +1,51 @@
+#### 启动Xcode
 
 ![](http://ww3.sinaimg.cn/large/a490147fjw1f5s9ge6xqvg20m80dwdxq.gif)
 
-Launch Xcode and create a Game project from the template for iOS. Name your game TicTacToe and make sure the language is set to Swift. During the project creation, Xcode creates a SKScene file that represents your initial view for your game along with a view controller file that initializes your game scene and handles presenting on screen when you launch the app. If you run the app now, you will see a Hello World label that displays to let you know that everything works out of the box. In addition, if you click the view, a space ship gets added to the location of the click. We don’t need the label or the space ship behavior anymore, so let’s remove that code. Switch to the GameScene.swift file, remove the code in didMoveToView and touchesBegan functions.
+启动XCode并且通过为iOS设计的模板创建一个游戏项目.　命名游戏为`TicTacToe`并且确认编程语言设定为Swift. 在项目的创建过程中，XCode创建了SKScene文件,　它展示了游戏的初始视图，连同一个视图控制器文件用于初始化你的游戏场景并且处理在应用程序启动的时候展现在屏幕上．如果你现在启动应用程序，你会看到Hello World标签，它让你所有的东西都立即可以使用了. 另外，如果你点击了视图, 一个宇宙飞船会增加在你点击的位置.　我们已经不再需要那个标签和宇宙飞船了，让我们移除那部分代码． 切换到`GameScene.swift`文件，移除`didMoveToView`和`touchesBegan`方法中的代码．
 
 ![](http://ww3.sinaimg.cn/large/a490147fjw1f5s9hnffbmj20m80angoe.jpg)
 
-Let’s spend a moment and highlight some of the features of the Scene Editor. The center of the view is the scene display, and the yellow outline around our tic tac toe board represents our view port that is visible for our game. We can can change the size of our view port or even add cameras that allow us to see more of the viewable parts of our game in realtime. In a “platformer”, we might create a large background image with enemy nodes scattered throughout the scene. We would use a camera node to advance across the scene to reveal new portions of the background over time. However, in this game, our view will be static around the board.
+让我们来花点时间并强调一些场景编辑器的特性． 视图的中心是显示了场景， 黄色的轮廓围绕着我们的tic tac toe游戏棋盘展现了我们的视口, 它让我们的游戏可见. 我们能改变游戏的视口或者增加摄像机, 让我们实时得看见更多游戏的内容. 在"platformer"终, 我们也可以创建一个很大的很多敌人点散列在场景终的背景图片. 我们也可以使用一个摄像机节点横跨整个场景随着时间去揭露更多新的背景部分. 然而, 对于这个游戏, 我们的视图将会时静止在棋盘附近.
 
 ![](http://ww4.sinaimg.cn/large/a490147fjw1f5s9hyl7ocj20m80dvwgg.jpg)
 
-At the bottom of the scene is our node editor. We can use this editor to add actions to the node or to select them more easily in our scene. For our purposes, we add a node to represent the board, labels, and placeholder nodes for each cell of the board. Last, notice that every node in our scene has a name, which we will use to reference in code.
+场景的底部是节点编辑器.  我们可以使用这个编辑器为节点增加功能或者在场景中更容易的选择他们. 我们用增加一个节点来表示游戏棋盘, 标签和每一格游戏棋盘的占位节点. 最后, 每一个在场景中的节点都有一个名字, 它用于在代码中引用回去.  
 
-I have provided the entire game on [Github](https://github.com/mrkeithelliott/tictactoe) so that you can follow along and investigate the areas that I am glossing over for the sake of time in this article.
+为了考虑写这篇文章的时间我已经将整个游戏项目提交到了[Github](https://github.com/mrkeithelliott/tictactoe), 所以你可以追随并且研究我(掩盖)带过的这个区域.
```

Comment:
so that you can follow along and investigate the areas that I am glossing over for the sake of time in this article. 这里的follow感觉可以当一个名词不做翻译。所以你可以follow这个项目研究我在本文中略过的内容